app/api/routes/dependencies.py

```diff
@@ -0,0 +1,50 @@
+from fastapi import Depends, HTTPException, status
+from fastapi.security import OAuth2PasswordBearer
+from models import TokenData, verify_token
+from user import users_db  
+from uuid import UUID
+
+oauth2_scheme = OAuth2PasswordBearer(tokenUrl="login")
+
+#Dependency to get the current user
+async def get_current_user(token: str = Depends(oauth2_scheme)) -> TokenData:
+    credentials_exception = HTTPException(
+        status_code=status.HTTP_401_UNAUTHORIZED,
+        detail="Could not validate credentials",
+        headers={"WWW-Authenticate": "Bearer"},
+    )
+    username = verify_token(token, credentials_exception)
```

Comment:
It's better always to wrap the logic with try except

Reply:
done.